### PR TITLE
K8s Neptune release branch

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Staged preview: 
* Release notes: https://docs.redis.com/staging/release-k8s-neptune/kubernetes/release-notes/k8s-6-2-18-3/
* Supported distros:  https://docs.redis.com/staging/release-k8s-neptune/kubernetes/reference/supported_k8s_distributions/
* Redis on Flash:  https://docs.redis.com/staging/release-k8s-neptune/kubernetes/re-clusters/redis-on-flash/

### CODE REVIEW

File: content/kubernetes/release-notes/k8s-6-2-18-3.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+* Fixed issue with updating credentials on Openshift when accessing the cluster externally with routes (RED-73251, RED-75329).
+
+## API changes
+
+The following fields were added to the [Redis Enterprise cluster (REC) API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md):
+
+* Added `.services.apiService.type` to allow configuration of the API service type.
+* Made `.redisOnFlashSpec` available by default.
+* Made `.ocspConfiguration` available by default for configuring OCSP stapling.
+* Made `encryptPkeys` available by default for configuring PEM encryption.
```
Comment (zcahana) [inline content/kubernetes/release-notes/k8s-6-2-18-3.md:59]: `. encryptPkeys` (add dot prefix for consistency with the above).

File: content/kubernetes/release-notes/k8s-6-2-18-3.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+The following fields were added to the [Redis Enterprise database (REDB) API:
```
Comment (zcahana) [inline content/kubernetes/release-notes/k8s-6-2-18-3.md:61]: Did you want to link the REDB API doc here?
If not, let's remove the `[` character.

File: content/kubernetes/release-notes/k8s-6-2-18-3.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+* Fixed issue with updating credentials on Openshift when accessing the cluster externally with routes (RED-73251, RED-75329).
+
+## API changes
+
+The following fields were added to the [Redis Enterprise cluster (REC) API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md):
+
+* Added `.services.apiService.type` to allow configuration of the API service type.
+* Made `.redisOnFlashSpec` available by default.
+* Made `.ocspConfiguration` available by default for configuring OCSP stapling.
+* Made `encryptPkeys` available by default for configuring PEM encryption.
```
Comment (kaitlynmichael) [inline content/kubernetes/release-notes/k8s-6-2-18-3.md:59]:
```suggestion
* Made `.encryptPkeys` available by default for configuring PEM encryption.
```

File: content/kubernetes/release-notes/k8s-6-2-18-3.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+The following fields were added to the [Redis Enterprise database (REDB) API:
```
Comment (kaitlynmichael) [inline content/kubernetes/release-notes/k8s-6-2-18-3.md:61]:
```suggestion
The following fields were added to the [Redis Enterprise database (REDB) API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_database_api.md):
```

**Review by heinrich-redislabs — APPROVED** [review]

**Pull request merged**